server/events/project_locker_test.go

```diff
@@ -139,3 +139,36 @@ func TestDefaultProjectLocker_TryLockUnlocked(t *testing.T) {
 	Ok(t, err)
 	mockLocker.VerifyWasCalledOnce().Unlock(lockKey)
 }
+
+func TestDefaultProjectLocker_RepoLocking(t *testing.T) {
```

Comment:
Can we test for both lock and no locking?

Reply:
added both case tests.

https://github.com/runatlantis/atlantis/pull/2700/commits/795328faf3e1d3025520a5fb5fb439d4259b13b9